src/gui/Shared/SpectrumAnalyser.h

```diff
@@ -15,6 +15,9 @@ class SpectrumAnalyser : public juce::Component,
     void setShouldShowPreEQ (bool shouldShow);
     void setShouldShowPostEQ (bool shouldShow);
 
+    std::atomic<float> minFrequencyHz;
+    std::atomic<float> maxFrequencyHz;
```

Comment:
I'm not 100% sure that these need to be atomic... We are reading the values on the "task" thread, but if we set the values in the main thread, before the task starts, then that should be thread-safe, and won't slow down the task thread as much.